### PR TITLE
Port regression_8100 test

### PR DESCRIPTION
**Pull request by dsemenets**
Port regression_8100 test

### CODE REVIEW

File: scripts/file_to_c.py
Comment (lorc) [inline scripts/file_to_c.py:1]: Congrats. You have broke cause-and-effect relationship.

You need to add this script **first** and then add a commit that relies on this script.

**Review by lorc — COMMENTED** [review]
For all ported files please mention the revision (tag) or commit id. This will help greatly when we will port any updates.

File: scripts/file_to_c.py
Comment (dsemenets) [reply scripts/file_to_c.py:1]: Commits rearranged, commit message fixed

**Comment by oleksiimoisieiev** [comment]
NAK please provide commit id in correct format

**Comment by oleksiimoisieiev** [comment]
LGTM 
`Reviewed-by: Oleksii Moisieiev <oleksii_moisieiev@epam.com>`
`Tested-by: Oleksii Moisieiev <oleksii_moisieiev@epam.com>`
Results on the Zephyr main branch:

```
===================================================================
TESTSUITE regression_8100 succeeded
End Test suite 8100

------ TESTSUITE SUMMARY START ------

SUITE PASS - 100.00% [regression_8100]: pass = 3, fail = 0, skip = 0, total = 3 duration = 4.464 seconds
 - PASS - [regression_8100.test_8101] duration = 3.844 seconds
 - PASS - [regression_8100.test_8102] duration = 0.276 seconds
 - PASS - [regression_8100.test_8103] duration = 0.344 seconds

------ TESTSUITE SUMMARY END ------

===================================================================
```

**Comment by dsemenets** [comment]
Updated tags

**Comment by oleksiimoisieiev** [comment]
@dsemenets Please update commit description. Commit format is wrong. No "." at the end of commit desctiption. please see doc https://github.com/xen-troops/xen-troops.github.io/blob/master/contribution.md

File: src/regression_8100.c [hunk 1]
```diff
@@ -0,0 +1,364 @@
...
+#include "xtest_helpers.h"
+
+#include <stdarg.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <ta_crypt.h>
+#include <tee_api_defines.h>
+#include <tee_api_types.h>
+//#include <tee_api_compat.h>
```
Comment (firscity) [inline src/regression_8100.c]: Please, no commented code.

**Review by firscity — APPROVED** [review]
With one comment:
`Acked-by: Dmytro Firsov <dmytro_firsov@epam.com>`

**Review by lorc — APPROVED** [review]
`Acked-by: Volodymyr Babchul <volodymyr_babchuk@epam.com>`

**Pull request merged**